src/devices/tuya.ts

```diff
@@ -8734,6 +8734,66 @@ const definitions: Definition[] = [
             ],
         },
     },
+    {
```

Comment:
Please move this definition to `neo.ts`

Reply:
Do I need to create a new pull for this?